gptel.el

```diff
@@ -74,6 +74,13 @@
   :group 'gptel
   :type 'string)
 
+(defcustom gptel-proxy-path ""
```

Comment:
Please rename to just `gptel-proxy`, for consistency with `gptel-host` (which is not `gptel-host-path`.)